node/p2p/blossomsub.go

```diff
@@ -144,20 +149,13 @@ func NewBlossomSubStreamer(
 
 	logger.Info("established peer id", zap.String("peer_id", h.ID().String()))
 
-	kademliaDHT := initDHT(
+	_ = initDHT(
 		ctx,
-		p2pConfig,
 		logger,
 		h,
 		false,
 		bootstrappers,
 	)
-	routingDiscovery := routing.NewRoutingDiscovery(kademliaDHT)
-	util.Advertise(ctx, routingDiscovery, getNetworkNamespace(p2pConfig.Network))
```

Comment:
Advertising while being a `SubStreamer` is not required, as the peer is not expected to participate in the mesh.